dexterity/membrane/profiles/uninstall_content/types.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0"?>
 <object name="portal_types">
```

Comment:
Should this file exist in the uninstall profile?

Reply:
sorry, resolved.